.circleci/config.yml

```diff
@@ -1,5 +1,23 @@
 version: 2.1
 jobs:
+  validator:
```

Comment:
As discussed, let's move it into the `testGenerator` task.

Reply:
👍 doing it now, thanks